src/app/custom-keyboard/_components/navigator/TotalCostWithNavigation.tsx

```diff
@@ -72,11 +72,11 @@ const UPDATE_NEXT_STEP_STATUS: UpdateStepType<'keyCap'> = {
 };
 
 export default function TotalCostWithNavigation({ accessToken }: TotalCostWithNavigationProps) {
-  const { data, isSuccess } = useQuery({
+  const { data } = useQuery({
```

Comment:
```suggestion
  const { data: dataName } = useQuery({
```
뽑아낸 데이터는 이름을 명시해주는게 정석적인 방법입니다